frame/support/procedural/src/benchmark.rs

```diff
@@ -705,36 +777,97 @@ fn expand_benchmark(
 				qself: None,
 				path: Path { leading_colon: None, segments: punct },
 			});
-
+			let pre_call = quote! {
+				let __call = Call::<#type_use_generics>::#expr_call;
+				let __benchmarked_call_encoded = #codec::Encode::encode(&__call);
```

Comment:
I couldn't find this code in the declarative macro, why is it being generated here?

Reply:
this stuff comes from around here it's a bit buried: https://github.com/paritytech/substrate/blob/14de95807ef53157360b09112454549077acef2b/frame/benchmarking/src/v1.rs#L443-L455

It's a lot easier to find if you cargo expand if I remember correctly

Reply:
If you're wondering why it's showing up in the diff from this PR, it's because I pulled it out from lower down in the expansion code because I needed to use it in the benchmark function expansion as well